src/spaces/grid_multi.jl

```diff
@@ -97,7 +97,8 @@ function remove_agent_from_space!(a::AbstractAgent, model::ABM{<:GridSpace})
     prev = abmspace(model).stored_ids[a.pos...]
     ai = findfirst(i -> i == a.id, prev)
     isnothing(ai) && error(lazy"Tried to remove $(a) from the space, but that agent is not on the space")
-    deleteat!(prev, ai)
+    prev[ai], prev[end] = prev[end], prev[ai]
```

Comment:
Perhaps you can add a comment here that points to this PR for future reference?

Reply:
added a small comment